libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -241,6 +241,31 @@ void AP_Camera_Backend::send_camera_settings(mavlink_channel_t chan) const
         NaN);               // focusLevel float, percentage from 0 to 100, NaN if unknown
 }
 
+// send camera capture status message to GCS
+void AP_Camera_Backend::send_camera_capture_status(mavlink_channel_t chan) const
+{
+    const float NaN = nanf("0x4152");
+
+    const uint32_t time_boot_ms = AP_HAL::millis();
+    const float image_interval = 0.0;     // Image capture interval (s)
+    const uint32_t recording_time_ms = 0; // Time since recording started (ms)
+    const float available_capacity = NaN; // Available storage capacity (MiB)
+    const uint8_t image_status = 0;       // Current status of image capturing (0: idle, 1: capture in progress, 2: interval set but idle, 3: interval set and capture in progress)
```

Comment:
we could set image_status = 2 if "time_interval_settings.num_remaining != 0".

We don't really have a good way to know if we have a "capture in progress".  Theoretically we might be able to use the contents of the camera_feedback structure but I doubt this would work for all cameras so probably best to leave it out for now.

Reply:
Updated `image_status` and `image_interval`. 

And also fixed the mis-ordering in the function call.